.gitlab/pipeline.yml

```
@@ -9,3 +9,4 @@ include:
   - local: '.gitlab/includes/clang14_cuda11_pipeline.yml'
   - local: '.gitlab/includes/performance_gcc13_pipeline.yml'
   - local: '.gitlab/includes/gcc12_hip5_pipeline.yml'
+  - local: '.gitlab/includes/nvhpc_mpi_pipeline.yml'
```

Maybe something in this direction:
```suggestion
  - local: '.gitlab/includes/nvhpc23_9_pipeline.yml'
```
?

---

I think you should leave the mpi variant, especially if the tests run successfully.